cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/RemoteDatasetFramework.java

```diff
@@ -295,10 +304,10 @@ private static Location createDeploymentJar(Class<?> clz, Location destination)
 
   // can be used directly if DatasetTypeMeta is known, like in create dataset by dataset ops executor service
   public <T extends DatasetType> T getDatasetType(DatasetTypeMeta implementationInfo,
```

Comment:
this should have a Javadoc. 
Also, why is this not a method in the interface? Is it only useful for remote dataset frameworks?